### PR TITLE
SLES-15 add checks and remediations.

### PR DESCRIPTION
**Pull request by brett060102**
The following are all limitted to adding @sles15
info to existing rules and where needed adding
multi_platform_sle to ansible/shared.yml

- SLES-15-020210 'Set Existing Passwords Minimum Age'
- SLES-15-030290 'Record Events that Modify the System'
- SLES-15-030450 'Record Any Attempts to Run chcon'
- SLES-15-030530 'Ensure auditd Collects Information on Kernel Module Loading and Unloading - finit_module'
- SLES-15-030540 'Ensure auditd Collects Information on Kernel Module Loading - init_module'
- SLES-15-030360 'Ensure auditd Collects Information on the Use of Privileged Commands - umount'
- SLES-15-030030 'Record Events that Modify User/Group Information - /etc/security/opasswd'
- SLES-15-010370 'Verify firewalld Enabled'
- SLES-15-040381 'Disable Kernel Parameter for IPv6 Forwarding'
- SLES-15-040321 'Disable Kernel Parameter for Accepting Source-Routed Packets on IPv6 Interfaces by Default'
- SLES-15-010359 'Verify that System Executables Have Root Ownership'
- SLES-15-010358 'Verify that System Executables Have Restrictive Permissions'
- SLES-15-010330 'Encrypt Partitions'
- SLES-15-020102 'Ensure Users Re-Authenticate for Privilege Escalation - sudo'

fix SLE-15 STIG in sysctl_net_ipv6_conf_default_accept_source_route,
should have been SLES-15-040321.

#### Description:

-SLES-15 add checks and remediations.

The following are all limited to adding @sles15
info to existing rules and where needed adding
multi_platform_sle to ansible/shared.yml

- SLES-15-020210 'Set Existing Passwords Minimum Age'
- SLES-15-030290 'Record Events that Modify the System'
- SLES-15-030450 'Record Any Attempts to Run chcon'
- SLES-15-030530 'Ensure auditd Collects Information on Kernel Module Loading and Unloading - finit_module'
- SLES-15-030540 'Ensure auditd Collects Information on Kernel Module Loading - init_module'
- SLES-15-030360 'Ensure auditd Collects Information on the Use of Privileged Commands - umount'
- SLES-15-030030 'Record Events that Modify User/Group Information - /etc/security/opasswd'
- SLES-15-010370 'Verify firewalld Enabled'
- SLES-15-040381 'Disable Kernel Parameter for IPv6 Forwarding'
- SLES-15-040321 'Disable Kernel Parameter for Accepting Source-Routed Packets on IPv6 Interfaces by Default'
- SLES-15-010359 'Verify that System Executables Have Root Ownership'
- SLES-15-010358 'Verify that System Executables Have Restrictive Permissions'
- SLES-15-010330 'Encrypt Partitions'
- SLES-15-020102 'Ensure Users Re-Authenticate for Privilege Escalation - sudo'

fix SLE-15 STIG in sysctl_net_ipv6_conf_default_accept_source_route,
should have been SLES-15-040321.

#### Rationale:

- SUSE SLES-15 CaC support

### CODE REVIEW

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_init/rule.yml [hunk 1]
```diff
@@ -28,20 +28,25 @@ identifiers:
...
     disa: CCI-000172
+    disa@sle15: CCI-000130,CCI-000169,CCI-002884
     hipaa: 164.308(a)(1)(ii)(D),164.308(a)(3)(ii)(A),164.308(a)(5)(ii)(C),164.312(a)(2)(i),164.312(b),164.312(d),164.312(e)
     nist: AU-2(d),AU-12(c),AC-6(9),CM-6(a)
+    nist@sle15: AU-3,AU-3.1,AU-12(a),AU-12.1(ii),AU-12.1(iv),MA-4(1)(a)
     nist-csf: DE.AE-3,DE.AE-5,DE.CM-1,DE.CM-3,DE.CM-7,ID.SC-4,PR.AC-3,PR.PT-1,PR.PT-4,RS.AN-1,RS.AN-4
     ospp: FAU_GEN.1.1.c
     pcidss: Req-10.2.7
     srg: SRG-OS-000471-GPOS-00216,SRG-OS-000477-GPOS-00222
+    srgid@sle15: SRG-OS-000037-GPOS-00015,SRG-OS-000062-GPOS-00031,SRG-OS-000392-GPOS-00172,SRG-OS-000462-GPOS-00206,SRG-OS-000471-GPOS-00215
```
Comment (vojtapolasek) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_init/rule.yml:45]:
```suggestion
    srg@sle15: SRG-OS-000037-GPOS-00015,SRG-OS-000062-GPOS-00031,SRG-OS-000392-GPOS-00172,SRG-OS-000462-GPOS-00206,SRG-OS-000471-GPOS-00215
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_usergroup_modification_opasswd/rule.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,rhcos4,ol7,ol8,rhel7,rhel8,rhv4,wrlinux1019,sle12
+prodtype: fedora,rhcos4,ol7,ol8,rhel7,rhel8,rhv4,wrlinux1019,sle12,sle15
```
Comment (vojtapolasek) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_usergroup_modification_opasswd/rule.yml:3]:
```suggestion
prodtype: fedora,rhcos4,ol7,ol8,rhel7,rhel8,rhv4,sle12,sle15,wrlinux1019
```

File: linux_os/guide/system/network/network-firewalld/firewalld_activation/service_firewalld_enabled/rule.yml [hunk 3]
```diff
@@ -35,6 +36,11 @@ references:
     cis-csc: 11,3,9
     cis@sle15: 3.5.1.4
     stigid@rhel8: RHEL-08-040100
+    stigid@sle15: SLES-15-010370
+    srg@sles15: SRG-OS-000298-GPOS-00116
```
Comment (vojtapolasek) [inline linux_os/guide/system/network/network-firewalld/firewalld_activation/service_firewalld_enabled/rule.yml:40]:
```suggestion
    srg@sle15: SRG-OS-000298-GPOS-00116
```

File: linux_os/guide/system/network/network-firewalld/firewalld_activation/service_firewalld_enabled/rule.yml [hunk 4]
```diff
@@ -35,6 +36,11 @@ references:
...
+    nist@sles15: AC-17(9)
```
Comment (vojtapolasek) [inline linux_os/guide/system/network/network-firewalld/firewalld_activation/service_firewalld_enabled/rule.yml:41]:
```suggestion
    nist@sle15: AC-17(9)
```

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_ownership_binary_dirs/rule.yml [hunk 5]
```diff
@@ -53,3 +57,4 @@ ocil: |-
     To find system executables that are not owned by <tt>root</tt>,
     run the following command for each directory <i>DIR</i> which contains system executables:
     <pre>$ sudo find <i>DIR/</i> \! -user root</pre>
+
```
Comment (vojtapolasek) [inline linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_ownership_binary_dirs/rule.yml:60]: I think this blank line is redundant.

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/ansible/shared.yml [hunk 6]
```diff
@@ -16,3 +16,4 @@
     mode: "go-w"
   with_items: "{{ world_writable_library_files.stdout_lines }}"
   when: world_writable_library_files.stdout_lines | length > 0
+
```
Comment (vojtapolasek) [inline linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/ansible/shared.yml:19]: I think the blank line is redundant.

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/rule.yml [hunk 7]
```diff
@@ -53,3 +57,4 @@ ocil: |-
     To find system executables that are group-writable or world-writable,
     run the following command for each directory <i>DIR</i> which contains system executables:
     <pre>$ sudo find -L <i>DIR</i> -perm /022 -type f</pre>
+
```
Comment (vojtapolasek) [inline linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/rule.yml:60]: redundant blank line

File: linux_os/guide/system/software/disk_partitioning/encrypt_partitions/rule.yml [hunk 8]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel7,rhel8,rhv4,rhcos4,sle12
+prodtype: ol7,ol8,rhel7,rhel8,rhv4,rhcos4,sle12,sle15
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/disk_partitioning/encrypt_partitions/rule.yml:3]:
```suggestion
prodtype: ol7,ol8,rhcos4,rhel7,rhel8,rhv4,sle12,sle15
```

File: linux_os/guide/system/software/sudo/sudo_require_authentication/rule.yml [hunk 9]
```diff
@@ -31,6 +32,10 @@ references:
     cobit5: DSS05.04,DSS05.10,DSS06.03,DSS06.10
     iso27001-2013: A.18.1.4,A.9.2.1,A.9.2.2,A.9.2.3,A.9.2.4,A.9.2.6,A.9.3.1,A.9.4.2,A.9.4.3
     cis-csc: 1,12,15,16,5
+    srg@sle15: SRG-OS-000373-GPOS-00156
+    stigid@sle15: SLES-15-020102
+    disa@sle15: CCI-002038
+
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/sudo/sudo_require_authentication/rule.yml:38]: redundant blank line

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Nice to see sle15 progress. Please see comments.

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_ownership_binary_dirs/rule.yml (hunk 5 above)
Comment (brett060102) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_ownership_binary_dirs/rule.yml]: agreed

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/ansible/shared.yml (hunk 6 above)
Comment (brett060102) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/ansible/shared.yml]: yep

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/rule.yml (hunk 7 above)
Comment (brett060102) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_binary_dirs/rule.yml]: yes

**Comment by brett060102** [comment]
@vojtapolasek Thank you very much. I think I have everything addressed.

File: linux_os/guide/system/software/sudo/sudo_require_authentication/rule.yml (hunk 9 above)
Comment (brett060102) [reply linux_os/guide/system/software/sudo/sudo_require_authentication/rule.yml:38]: should be gone now.

**Comment by vojtapolasek** [comment]
Thank you, merging.

**Review by vojtapolasek — APPROVED** [review]

**Pull request merged**